src/Server.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\Ssofact\Server.
+ */
+
+namespace Netzstrategen\Ssofact;
+
+class Server {
+
+  /**
+   * Checks if the email is already registered.
+   *
+   * @param string email
+   *
+   * @return array
+   */
+  public static function checkIsEmailRegistered($email) {
+    $api_url = 'https://stage-login.stimme.de/REST/services/authenticate/user/IsEmailRegistered';
```

Comment:
looks suitable for a class constant

Reply:
Ideally split the domain from the endpoint as in the Drupal implementation: https://github.com/netzstrategen/drupal-ssofact/pull/2/files#diff-49da63fb6d6ddfc7b78205ca77633e0cR24

The domain will need to be configurable (e.g., via constant in wp-config.php).

The protocol is always HTTPS.